### DISCUSSION TITLE
这是我的战争资源失效了，还有版本也很老了。

### DISCUSSION THREAD

**Discussion by 972933728** [Gamuxorg/bbs#233] [投递和请求游戏]
现在的版本是6点几，站内的那个是4点几的并且它资源失效了不能下载。
它还不能全屏（过去我下载过一次，体验很不好。）

**Comment by hyh19962008** [comment]
站内的链接暂时还没更新，可以在这里下旧版 https://de.linuxgame.cn/
新版稍后补上

**Comment by hyh19962008** [comment]
新版没有资源……
试试能不能wine吧
